app/models/session.rb

```diff
@@ -94,6 +94,10 @@ def self.filter(data={})
       sessions = filter_by_time_range(sessions, data[:time_from], data[:time_to])
     end
 
+    if sessions
```

Comment:
Why do we need the `if`?

Reply:
Some tests were failing cause the query at the beginning of filter returned nil:
```
    sessions = order("sessions.created_at DESC")
    .where("contribute = true OR sessions.id in (?)", data[:session_ids])
    .joins(:user)
```
I susspect this can only happen it tests, but can't be sure so i added this `if`.

Reply:
I'd prefer if you took a bit of time and investigated not having the `if`. This method is already complex, let's try to avoid adding more on top